api/deviceflow.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+type ApproveDeviceFlowRequest struct {
+	UserCode string `json:"userCode" example:"BDSD-HQMK"`
+}
+
+type StartDeviceFlowRequest struct {
+	ClientID string `form:"client_id"`
+}
+
+type DeviceFlowResponse struct {
+	DeviceCode          string `json:"deviceCode" example:"NGU4QWFiNjQ5YmQwNG3YTdmZMEyNzQ3YzQ1YSA" note:"a long string the device will use to exchange for an access token"`
+	VerificationURI     string `json:"verificationURI" example:"https://infrahq.com/device" note:"This is the URL the user needs to enter into their browser to start logging in"`
+	UserCode            string `json:"userCode" example:"BDSD-HQMK" note:"This is the text the user will enter at the URL above"`
```

Comment:
```suggestion
	UserCode            string `json:"userCode" example:"BDSD-HQMK" note:"This is the text the user will enter at the verification URI"`
```

In case the order changes.